x-pack/plugins/observability/public/components/shared/exploratory_view/series_editor/columns/series_filter.tsx

```diff
@@ -44,12 +44,28 @@ export function SeriesFilter({ series, seriesConfig, seriesId }: Props) {
       };
     });
 
+  const hasUrlFilter = useMemo(() => {
+    return (
+      seriesConfig.filterFields.find((field) => {
```

Comment:
Could `filterFields.some()` be more suitable here since we're not really interested in the specific match?

Reply:
Yep, would be slightly more succinct 👍